test/parallel/test-assert.js

```diff
@@ -409,6 +409,11 @@ assert.throws(() => {
 }, /Got unwanted exception: user message/,
               'a.doesNotThrow ignores user message');
 
+assert.throws(() => {
+  assert.doesNotThrow(makeBlock(thrower, Error), 'user message');
+}, /Got unwanted exception: user message\ntest/,
+              'a.doesNotThrow ignores message of the unwanted exception');
```

Comment:
I think what @jasnell mean is that the opening `'` should be directly below the opening `/`. (Or just drop the extra third argument here, I think I would prefer that.)